### PR TITLE
Fix/155013 cluster recovery

### PR DESCRIPTION
**Pull request by dmitriyisaev**

### CODE REVIEW

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

File: app/controllers/MainController.scala [hunk 1]
```diff
@@ -49,8 +49,8 @@ class MainController @Inject() (actorSystem: ActorSystem,
     val dslinkStats = getDSLinkCounts
     val uplinkStats = getUplinkCounts
     for {
-      down <- dslinkStats
-      up <- uplinkStats
+      down <- dslinkStats.recover{case e => DSLinkStats(Map.empty)}
+      up <- uplinkStats.recover{case e => DSLinkStats(Map.empty)}
```
Comment (uralian) [inline app/controllers/MainController.scala]: perhaps `NonFatal(e)` instead of just `e`

**Review by uralian — APPROVED** [review]

File: app/controllers/MainController.scala (hunk 1 above)
Comment (dmitriyisaev) [reply app/controllers/MainController.scala]: Thanks, fixed.

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Pull request merged**